compiler/z/codegen/OMRInstOpCodeProperties.hpp

```diff
@@ -0,0 +1,15852 @@
+   /* .properties  = */ S390OpProp_None
+   },
+
+   {
+   /* .mnemonic    = */ OMR::InstOpCode::DC2,
+   /* .name        = */ "DC2",
+   /* .description = */ "DC2",
+   /* .opcode[0]   = */ 0x00,
+   /* .opcode[1]   = */ 0x00,
+   /* .format      = */ DC_FORMAT,
+   /* .minimumALS  = */ TR_S390ProcessorInfo::TR_UnknownArchitecture,
+   /* .properties  = */ S390OpProp_None
+   },
+
+   {
+   /* .mnemonic    = */ OMR::InstOpCode::DCB,
+   /* .name        = */ "DCB",
+   /* .description = */ "Debug Counter Bump",
+   /* .opcode[0]   = */ 0x00,
+   /* .opcode[1]   = */ 0x00,
+   /* .format      = */ PSEUDO,
+   /* .minimumALS  = */ TR_S390ProcessorInfo::TR_UnknownArchitecture,
+   /* .properties  = */ S390OpProp_SetsSignFlag |
+                        S390OpProp_SetsOverflowFlag |
+                        S390OpProp_IsLoad |
+                        S390OpProp_IsStore
+   },
+
+   {
+   /* .mnemonic    = */ OMR::InstOpCode::DEPEND,
+   /* .name        = */ "DEPEND",
+   /* .description = */ "Someplace to hang dependencies",
+   /* .opcode[0]   = */ 0x00,
+   /* .opcode[1]   = */ 0x00,
+   /* .format      = */ PSEUDO,
+   /* .minimumALS  = */ TR_S390ProcessorInfo::TR_UnknownArchitecture,
+   /* .properties  = */ S390OpProp_None
+   },
+
+   {
+   /* .mnemonic    = */ OMR::InstOpCode::DIDTR,
+   /* .name        = */ "DIDTR",
+   /* .description = */ "Divide to Integer (DFP64)",
+   /* .opcode[0]   = */ 0xB4,
+   /* .opcode[1]   = */ 0xC6,
+   /* .format      = */ RRF3_FORMAT,
+   /* .minimumALS  = */ TR_S390ProcessorInfo::TR_UnknownArchitecture,
+   /* .properties  = */ S390OpProp_DoubleFP |
+                        S390OpProp_Is64Bit |
+                        S390OpProp_SetsOverflowFlag |
+                        S390OpProp_SetsOperand3
+   },
+
+   {
+   /* .mnemonic    = */ OMR::InstOpCode::DIRECTIVE,
+   /* .name        = */ "DIRECTIVE",
+   /* .description = */ "WCode DIR related",
+   /* .opcode[0]   = */ 0x00,
+   /* .opcode[1]   = */ 0x00,
+   /* .format      = */ PSEUDO,
+   /* .minimumALS  = */ TR_S390ProcessorInfo::TR_UnknownArchitecture,
+   /* .properties  = */ S390OpProp_None
+   },
+
+   {
+   /* .mnemonic    = */ OMR::InstOpCode::DS,
+   /* .name        = */ "DS",
+   /* .description = */ "DS",
+   /* .opcode[0]   = */ 0x00,
+   /* .opcode[1]   = */ 0x00,
+   /* .format      = */ PSEUDO,
+   /* .minimumALS  = */ TR_S390ProcessorInfo::TR_UnknownArchitecture,
+   /* .properties  = */ S390OpProp_None
+   },
+
+   {
+   /* .mnemonic    = */ OMR::InstOpCode::FENCE,
+   /* .name        = */ "FENCE",
+   /* .description = */ "Fence",
+   /* .opcode[0]   = */ 0x00,
+   /* .opcode[1]   = */ 0x00,
+   /* .format      = */ PSEUDO,
+   /* .minimumALS  = */ TR_S390ProcessorInfo::TR_UnknownArchitecture,
+   /* .properties  = */ S390OpProp_None
+   },
+
+   {
+   /* .mnemonic    = */ OMR::InstOpCode::KMGCM,
+   /* .name        = */ "KMGCM",
+   /* .description = */ "cipher message with galois counter mode",
+   /* .opcode[0]   = */ 0xB9,
```

Comment:
Given that KMGCM is a pseudo instruction now, we should make the opcode 0x00.

Reply:
Good catch! Fixed in 58f38c1

Reply:
There's one use of KMGCM in the OpenJ9 side. We should rename it there, too.